### ISSUE TITLE
No asset found for target: `x86_64-unknown-linux-gnu` when the self upgrade function of a binary is executed

### ISSUE THREAD

**Issue by Jocomol** [r-darwish/topgrade#633]
**What did you expect to happen?**
Normal self upgrade of the binary.

**What actually happened?**
The step "Self update" failed with "Self update error: ReleaseError: No asset found for target: `armv7-unknown-linux-gnueabihf`" this happens on al machines where I use the binary, AUR version works fine. The binaries where working fine before today.

**Additional details:**
- Raspbian GNU/Linux 10 (buster)
- Installed as a binary

<details><pre>
pi@pihole:~ $ topgrade -v
 2021-02-15T08:15:24.641Z DEBUG topgrade::config > Configuration at /home/pi/.config/topgrade.toml
 2021-02-15T08:15:24.642Z DEBUG topgrade::config > Path ~/* expanded to /home/pi/*
 2021-02-15T08:15:24.642Z DEBUG topgrade::config > Loaded configuration: ConfigFile { pre_commands: None, post_commands: None, commands: Some({"apt autoremove": "sudo apt autoremove -y"}), git_repos: None, predefined_git_repos: None, disable: None, ignore_failures: None, remote_topgrades: None, remote_topgrade_path: None, ssh_arguments: None, git_arguments: None, tmux_arguments: None, set_title: None, assume_yes: None, yay_arguments: None, no_retry: None, run_in_tmux: None, cleanup: None, notify_each_step: None, accept_all_windows_updates: None, bashit_branch: None, only: None, composer: None, brew: None, linux: None, git: Some(Git { max_concurrency: None, arguments: None, repos: Some(["/home/pi/*"]), pull_predefined: None }), windows: None, vagrant: None }
 2021-02-15T08:15:24.643Z DEBUG topgrade::utils  > Cannot find "notify-send"
 2021-02-15T08:15:24.643Z DEBUG topgrade         > Version: 6.5.1
 2021-02-15T08:15:24.643Z DEBUG topgrade         > OS: armv7-unknown-linux-gnueabihf
 2021-02-15T08:15:24.643Z DEBUG topgrade         > Args { inner: ["topgrade", "-v"] }
 2021-02-15T08:15:24.643Z DEBUG topgrade         > Binary path: Ok("/home/pi/.local/bin/topgrade")
 2021-02-15T08:15:24.643Z DEBUG topgrade         > Self Update: true
 2021-02-15T08:15:24.643Z DEBUG topgrade::utils  > Detected "/usr/bin/git" as "git"
 2021-02-15T08:15:24.643Z DEBUG topgrade::utils  > Cannot find "doas"
 2021-02-15T08:15:24.644Z DEBUG topgrade::utils  > Detected "/usr/bin/sudo" as "sudo"

―― 09:15:24 - Self update ――――――――――――――――――――――――――――――――――――――――――――――――――――――
Self update error: ReleaseError: No asset found for target: `armv7-unknown-linux-gnueabihf`
 2021-02-15T08:15:24.853Z DEBUG topgrade::utils  > Cannot find "pwsh"
 2021-02-15T08:15:24.853Z DEBUG topgrade::utils  > Cannot find "powershell"
 2021-02-15T08:15:24.853Z DEBUG topgrade::runner > Step "System update"

―― 09:15:24 - System update ――――――――――――――――――――――――――――――――――――――――――――――――――――
 2021-02-15T08:15:24.855Z DEBUG topgrade::utils  > Cannot find "apt-fast"
 2021-02-15T08:15:24.855Z DEBUG topgrade::executor > Running "/usr/bin/sudo" "/usr/bin/apt-get" "update"
Hit:1 http://raspbian.raspberrypi.org/raspbian buster InRelease
Hit:2 http://archive.raspberrypi.org/debian buster InRelease   
Hit:3 http://packages.microsoft.com/repos/code stable InRelease
Reading package lists... Done
 2021-02-15T08:15:31.331Z DEBUG topgrade::executor > Running "/usr/bin/sudo" "/usr/bin/apt-get" "dist-upgrade"
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Calculating upgrade... Done
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
 2021-02-15T08:15:33.992Z DEBUG topgrade::runner   > Step "etc-update"
 2021-02-15T08:15:33.993Z DEBUG topgrade::runner   > Step "Brew"
 2021-02-15T08:15:33.993Z DEBUG topgrade::runner   > Step "yadm"
 2021-02-15T08:15:33.994Z DEBUG topgrade::runner   > Step "nix"
 2021-02-15T08:15:33.994Z DEBUG topgrade::runner   > Step "home-manager"
 2021-02-15T08:15:33.994Z DEBUG topgrade::runner   > Step "asdf"
 2021-02-15T08:15:33.995Z DEBUG topgrade::utils    > Path "/home/pi/.config/emacs" doesn't exist
 2021-02-15T08:15:33.995Z DEBUG topgrade::utils    > Path "/home/pi/.emacs.d" doesn't exist
 2021-02-15T08:15:33.995Z DEBUG topgrade::steps::git > /home/pi/.doom.d does not exists
 2021-02-15T08:15:33.995Z DEBUG topgrade::steps::git > /home/pi/.vim does not exists
 2021-02-15T08:15:33.996Z DEBUG topgrade::steps::git > /home/pi/.config/nvim does not exists
 2021-02-15T08:15:33.996Z DEBUG topgrade::steps::git > /home/pi/.zshrc does not exists
 2021-02-15T08:15:33.996Z DEBUG topgrade::steps::git > /home/pi/.tmux does not exists
 2021-02-15T08:15:33.996Z DEBUG topgrade::steps::git > /home/pi/.config/fish does not exists
 2021-02-15T08:15:33.996Z DEBUG topgrade::steps::git > /home/pi/.config/openbox does not exists
 2021-02-15T08:15:33.997Z DEBUG topgrade::steps::git > /home/pi/.config/bspwm does not exists
 2021-02-15T08:15:33.997Z DEBUG topgrade::steps::git > /home/pi/.config/i3 does not exists
 2021-02-15T08:15:33.997Z DEBUG topgrade::steps::git > /home/pi/.config/sway does not exists
 2021-02-15T08:15:33.998Z DEBUG topgrade::steps::git > /home/pi/.bash_aliases is a file. Checking /home/pi
 2021-02-15T08:15:33.998Z DEBUG topgrade::steps::git > Checking if /home/pi is a git repository
 2021-02-15T08:15:34.004Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2021-02-15T08:15:34.004Z DEBUG topgrade::steps::git > /home/pi/.bash_history is a file. Checking /home/pi
 2021-02-15T08:15:34.005Z DEBUG topgrade::steps::git > Checking if /home/pi is a git repository
 2021-02-15T08:15:34.010Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2021-02-15T08:15:34.011Z DEBUG topgrade::steps::git > /home/pi/.bash_logout is a file. Checking /home/pi
 2021-02-15T08:15:34.011Z DEBUG topgrade::steps::git > Checking if /home/pi is a git repository
 2021-02-15T08:15:34.017Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2021-02-15T08:15:34.017Z DEBUG topgrade::steps::git > /home/pi/.bashrc is a file. Checking /home/pi
 2021-02-15T08:15:34.018Z DEBUG topgrade::steps::git > Checking if /home/pi is a git repository
 2021-02-15T08:15:34.024Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2021-02-15T08:15:34.024Z DEBUG topgrade::steps::git > Checking if /home/pi/.byobu is a git repository
 2021-02-15T08:15:34.030Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2021-02-15T08:15:34.031Z DEBUG topgrade::steps::git > Checking if /home/pi/.cache is a git repository
 2021-02-15T08:15:34.037Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2021-02-15T08:15:34.037Z DEBUG topgrade::steps::git > Checking if /home/pi/.config is a git repository
 2021-02-15T08:15:34.043Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2021-02-15T08:15:34.044Z DEBUG topgrade::steps::git > Checking if /home/pi/.gnupg is a git repository
 2021-02-15T08:15:34.050Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2021-02-15T08:15:34.051Z DEBUG topgrade::steps::git > Checking if /home/pi/.local is a git repository
 2021-02-15T08:15:34.057Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2021-02-15T08:15:34.057Z DEBUG topgrade::steps::git > /home/pi/.preise.py.swp is a file. Checking /home/pi
 2021-02-15T08:15:34.058Z DEBUG topgrade::steps::git > Checking if /home/pi is a git repository
 2021-02-15T08:15:34.064Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2021-02-15T08:15:34.064Z DEBUG topgrade::steps::git > /home/pi/.profile is a file. Checking /home/pi
 2021-02-15T08:15:34.065Z DEBUG topgrade::steps::git > Checking if /home/pi is a git repository
 2021-02-15T08:15:34.071Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2021-02-15T08:15:34.071Z DEBUG topgrade::steps::git > /home/pi/.python_history is a file. Checking /home/pi
 2021-02-15T08:15:34.071Z DEBUG topgrade::steps::git > Checking if /home/pi is a git repository
 2021-02-15T08:15:34.078Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2021-02-15T08:15:34.078Z DEBUG topgrade::steps::git > /home/pi/.selected_editor is a file. Checking /home/pi
 2021-02-15T08:15:34.078Z DEBUG topgrade::steps::git > Checking if /home/pi is a git repository
 2021-02-15T08:15:34.084Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2021-02-15T08:15:34.085Z DEBUG topgrade::steps::git > Checking if /home/pi/.ssh is a git repository
 2021-02-15T08:15:34.091Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2021-02-15T08:15:34.091Z DEBUG topgrade::steps::git > /home/pi/.viminfo is a file. Checking /home/pi
 2021-02-15T08:15:34.091Z DEBUG topgrade::steps::git > Checking if /home/pi is a git repository
 2021-02-15T08:15:34.097Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2021-02-15T08:15:34.098Z DEBUG topgrade::steps::git > Checking if /home/pi/.w3m is a git repository
 2021-02-15T08:15:34.104Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2021-02-15T08:15:34.104Z DEBUG topgrade::steps::git > /home/pi/.wget-hsts is a file. Checking /home/pi
 2021-02-15T08:15:34.104Z DEBUG topgrade::steps::git > Checking if /home/pi is a git repository
 2021-02-15T08:15:34.110Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2021-02-15T08:15:34.111Z DEBUG topgrade::steps::git > Checking if /home/pi/PADD is a git repository
 2021-02-15T08:15:34.117Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/home/pi/PADD\n", stderr: "" }
 2021-02-15T08:15:34.117Z DEBUG topgrade::runner     > Step "Git repositories"

―― 09:15:34 - Git repositories ―――――――――――――――――――――――――――――――――――――――――――――――――
 2021-02-15T08:15:34.129Z TRACE topgrade::executor   > Output of "/usr/bin/git" "remote" "show": Output { status: ExitStatus(ExitStatus(0)), stdout: "origin\n", stderr: "" }
 2021-02-15T08:15:34.137Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(ExitStatus(0)), stdout: "ef5acdc7e09389ce4212726d43f084c25a82b53a\n", stderr: "" }
Pulling /home/pi/PADD
 2021-02-15T08:15:35.023Z TRACE topgrade::executor   > Output of "/usr/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(ExitStatus(0)), stdout: "ef5acdc7e09389ce4212726d43f084c25a82b53a\n", stderr: "" }
Up-to-date /home/pi/PADD
 2021-02-15T08:15:35.024Z DEBUG topgrade::runner     > Step "zr"
 2021-02-15T08:15:35.024Z DEBUG topgrade::runner     > Step "antibody"
 2021-02-15T08:15:35.024Z DEBUG topgrade::runner     > Step "antigen"
 2021-02-15T08:15:35.025Z DEBUG topgrade::runner     > Step "zplug"
 2021-02-15T08:15:35.025Z DEBUG topgrade::runner     > Step "zinit"
 2021-02-15T08:15:35.025Z DEBUG topgrade::runner     > Step "zim"
 2021-02-15T08:15:35.025Z DEBUG topgrade::runner     > Step "oh-my-zsh"
 2021-02-15T08:15:35.025Z DEBUG topgrade::runner     > Step "fisher"
 2021-02-15T08:15:35.026Z DEBUG topgrade::runner     > Step "bash-it"
 2021-02-15T08:15:35.026Z DEBUG topgrade::runner     > Step "oh-my-fish"
 2021-02-15T08:15:35.026Z DEBUG topgrade::runner     > Step "tmux"
 2021-02-15T08:15:35.026Z DEBUG topgrade::runner     > Step "TLDR"
 2021-02-15T08:15:35.026Z DEBUG topgrade::runner     > Step "pearl"
 2021-02-15T08:15:35.026Z DEBUG topgrade::runner     > Step "SDKMAN!"
 2021-02-15T08:15:35.027Z DEBUG topgrade::utils      > Detected "/bin/bash" as "bash"
 2021-02-15T08:15:35.027Z DEBUG topgrade::runner     > Step "vscode"
 2021-02-15T08:15:35.027Z DEBUG topgrade::runner     > Step "apm"
 2021-02-15T08:15:35.027Z DEBUG topgrade::runner     > Step "rustup"
 2021-02-15T08:15:35.027Z DEBUG topgrade::runner     > Step ".NET"
 2021-02-15T08:15:35.028Z DEBUG topgrade::runner     > Step "choosenim"
 2021-02-15T08:15:35.028Z DEBUG topgrade::runner     > Step "cargo"
 2021-02-15T08:15:35.028Z DEBUG topgrade::runner     > Step "Flutter"
 2021-02-15T08:15:35.028Z DEBUG topgrade::runner     > Step "Go"
 2021-02-15T08:15:35.028Z DEBUG topgrade::runner     > Step "Emacs"
 2021-02-15T08:15:35.029Z DEBUG topgrade::runner     > Step "opam"
 2021-02-15T08:15:35.029Z DEBUG topgrade::runner     > Step "vcpkg"
 2021-02-15T08:15:35.029Z DEBUG topgrade::runner     > Step "pipx"
 2021-02-15T08:15:35.029Z DEBUG topgrade::runner     > Step "stack"
 2021-02-15T08:15:35.030Z DEBUG topgrade::runner     > Step "tlmgr"
 2021-02-15T08:15:35.030Z DEBUG topgrade::runner     > Step "myrepos"
 2021-02-15T08:15:35.030Z DEBUG topgrade::runner     > Step "jetpack"
 2021-02-15T08:15:35.030Z DEBUG topgrade::runner     > Step "vim"
 2021-02-15T08:15:35.030Z DEBUG topgrade::utils      > Detected "/usr/bin/vim" as "vim"
 2021-02-15T08:15:35.044Z TRACE topgrade::executor   > Output of "/usr/bin/vim" "--version": Output { status: ExitStatus(ExitStatus(0)), stdout: "VIM - Vi IMproved 8.1 (2018 May 18, compiled Jun 15 2019 16:41:15)\nIncluded patches: 1-875, 878, 884, 948, 1046, 1365-1368, 1382, 1401\nModified by team+vim@tracker.debian.org\nCompiled by team+vim@tracker.debian.org\nHuge version without GUI.  Features included (+) or not (-):\n+acl               +extra_search      +mouse_netterm     +tag_old_static\n+arabic            +farsi             +mouse_sgr         -tag_any_white\n+autocmd           +file_in_path      -mouse_sysmouse    -tcl\n+autochdir         +find_in_path      +mouse_urxvt       +termguicolors\n-autoservername    +float             +mouse_xterm       +terminal\n-balloon_eval      +folding           +multi_byte        +terminfo\n+balloon_eval_term -footer            +multi_lang        +termresponse\n-browse            +fork()            -mzscheme          +textobjects\n++builtin_terms    +gettext           +netbeans_intg     +textprop\n+byte_offset       -hangul_input      +num64             +timers\n+channel           +iconv             +packages          +title\n+cindent           +insert_expand     +path_extra        -toolbar\n-clientserver      +job               -perl              +user_commands\n-clipboard         +jumplist          +persistent_undo   +vartabs\n+cmdline_compl     +keymap            +postscript        +vertsplit\n+cmdline_hist      +lambda            +printer           +virtualedit\n+cmdline_info      +langmap           +profile           +visual\n+comments          +libcall           -python            +visualextra\n+conceal           +linebreak         -python3           +viminfo\n+cryptv            +lispindent        +quickfix          +vreplace\n+cscope            +listcmds          +reltime           +wildignore\n+cursorbind        +localmap          +rightleft         +wildmenu\n+cursorshape       -lua               -ruby              +windows\n+dialog_con        +menu              +scrollbind        +writebackup\n+diff              +mksession         +signs             -X11\n+digraphs          +modify_fname      +smartindent       -xfontset\n-dnd               +mouse             +startuptime       -xim\n-ebcdic            -mouseshape        +statusline        -xpm\n+emacs_tags        +mouse_dec         -sun_workshop      -xsmp\n+eval              +mouse_gpm         +syntax            -xterm_clipboard\n+ex_extra          -mouse_jsbterm     +tag_binary        -xterm_save\n   system vimrc file: \"$VIM/vimrc\"\n     user vimrc file: \"$HOME/.vimrc\"\n 2nd user vimrc file: \"~/.vim/vimrc\"\n      user exrc file: \"$HOME/.exrc\"\n       defaults file: \"$VIMRUNTIME/defaults.vim\"\n  fall-back for $VIM: \"/usr/share/vim\"\nCompilation: gcc -c -I. -Iproto -DHAVE_CONFIG_H   -Wdate-time  -g -O2 -fdebug-prefix-map=/build/vim-jkgW07/vim-8.1.0875=. -fstack-protector-strong -Wformat -Werror=format-security -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1       \nLinking: gcc   -Wl,-z,relro -Wl,-z,now -Wl,--as-needed -o vim        -lm -ltinfo -lnsl  -lselinux -lacl -lattr -lgpm -ldl           \n", stderr: "" }
 2021-02-15T08:15:35.045Z DEBUG topgrade::runner     > Step "Neovim"
 2021-02-15T08:15:35.045Z DEBUG topgrade::runner     > Step "voom"
 2021-02-15T08:15:35.045Z DEBUG topgrade::runner     > Step "npm"
 2021-02-15T08:15:35.046Z DEBUG topgrade::runner     > Step "yarn"
 2021-02-15T08:15:35.046Z DEBUG topgrade::runner     > Step "deno"
 2021-02-15T08:15:35.046Z DEBUG topgrade::runner     > Step "composer"
 2021-02-15T08:15:35.046Z DEBUG topgrade::runner     > Step "krew"
 2021-02-15T08:15:35.046Z DEBUG topgrade::runner     > Step "gem"
 2021-02-15T08:15:35.047Z DEBUG topgrade::runner     > Step "sheldon"
 2021-02-15T08:15:35.047Z DEBUG topgrade::runner     > Step "rtcl"
 2021-02-15T08:15:35.047Z DEBUG topgrade::runner     > Step "gcloud"
 2021-02-15T08:15:35.047Z DEBUG topgrade::runner     > Step "Flatpak"
 2021-02-15T08:15:35.048Z DEBUG topgrade::runner     > Step "snap"
 2021-02-15T08:15:35.048Z DEBUG topgrade::runner     > Step "apt autoremove"

―― 09:15:35 - apt autoremove ―――――――――――――――――――――――――――――――――――――――――――――――――――
 2021-02-15T08:15:35.050Z DEBUG topgrade::executor   > Running "/bin/bash" "-c" "sudo apt autoremove -y"
Reading package lists... Done
Building dependency tree       
Reading state information... Done
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
 2021-02-15T08:15:40.591Z DEBUG topgrade::runner     > Step "pihole"
 2021-02-15T08:15:40.591Z DEBUG topgrade::utils      > Detected "/usr/local/bin/pihole" as "pihole"
 2021-02-15T08:15:40.591Z DEBUG topgrade::utils      > Path "/opt/pihole/update.sh" exists

―― 09:15:40 - pihole ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2021-02-15T08:15:40.593Z DEBUG topgrade::executor   > Running "/usr/bin/sudo" "/usr/local/bin/pihole" "-up"
  [i] Checking for updates...
  [i] Pi-hole Core:	up to date
  [i] Web Interface:	up to date
  [i] FTL:		up to date

  [✓] Everything is up to date!
 2021-02-15T08:15:43.335Z DEBUG topgrade::runner     > Step "Firmware upgrades"
 2021-02-15T08:15:43.336Z DEBUG topgrade::runner     > Step "Restarts"

―― 09:15:43 - Summary ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
System update: OK
etc-update: SKIPPED: Cannot find "etc-update" in PATH
Brew: SKIPPED: Cannot find "brew" in PATH
yadm: SKIPPED: Cannot find "yadm" in PATH
nix: SKIPPED: Cannot find "nix" in PATH
home-manager: SKIPPED: Cannot find "home-manager" in PATH
asdf: SKIPPED: Cannot find "asdf" in PATH
Git repositories: OK
zr: SKIPPED: Cannot find "zsh" in PATH
antibody: SKIPPED: Cannot find "zsh" in PATH
antigen: SKIPPED: Cannot find "zsh" in PATH
zplug: SKIPPED: Cannot find "zsh" in PATH
zinit: SKIPPED: Cannot find "zsh" in PATH
zim: SKIPPED: Cannot find "zsh" in PATH
oh-my-zsh: SKIPPED: Cannot find "zsh" in PATH
fisher: SKIPPED: Cannot find "fish" in PATH
bash-it: SKIPPED: Path "/home/pi/.bash_it" doesn't exist
oh-my-fish: SKIPPED: Cannot find "fish" in PATH
tmux: SKIPPED: Path "/home/pi/.tmux/plugins/tpm/bin/update_plugins" doesn't exist
TLDR: SKIPPED: Cannot find "tldr" in PATH
pearl: SKIPPED: Cannot find "pearl" in PATH
SDKMAN!: SKIPPED: Path "/home/pi/.sdkman/bin/sdkman-init.sh" doesn't exist
vscode: SKIPPED: Cannot find "code" in PATH
apm: SKIPPED: Cannot find "apm" in PATH
rustup: SKIPPED: Cannot find "rustup" in PATH
.NET: SKIPPED: Cannot find "dotnet" in PATH
choosenim: SKIPPED: Cannot find "choosenim" in PATH
cargo: SKIPPED: Cannot find "cargo" in PATH
Flutter: SKIPPED: Cannot find "flutter" in PATH
Go: SKIPPED: Cannot find "go" in PATH
Emacs: SKIPPED: Cannot find "emacs" in PATH
opam: SKIPPED: Cannot find "opam" in PATH
vcpkg: SKIPPED: Cannot find "vcpkg" in PATH
pipx: SKIPPED: Cannot find "pipx" in PATH
stack: SKIPPED: Cannot find "stack" in PATH
tlmgr: SKIPPED: tlmgr must be explicity enabled in the configuration to run in Linux
myrepos: SKIPPED: Cannot find "mr" in PATH
jetpack: SKIPPED: Cannot find "jetpack" in PATH
vim: SKIPPED: Path "/home/pi/.vim/vimrc" doesn't exist
Neovim: SKIPPED: Cannot find "nvim" in PATH
voom: SKIPPED: Cannot find "voom" in PATH
npm: SKIPPED: Cannot find "npm" in PATH
yarn: SKIPPED: Cannot find "yarn" in PATH
deno: SKIPPED: Cannot find "deno" in PATH
composer: SKIPPED: Cannot find "composer" in PATH
krew: SKIPPED: Cannot find "kubectl-krew" in PATH
gem: SKIPPED: Cannot find "gem" in PATH
sheldon: SKIPPED: Cannot find "sheldon" in PATH
rtcl: SKIPPED: Cannot find "rupdate" in PATH
gcloud: SKIPPED: Cannot find "gcloud" in PATH
Flatpak: SKIPPED: Cannot find "flatpak" in PATH
snap: SKIPPED: Cannot find "snap" in PATH
apt autoremove: OK
pihole: OK
Firmware upgrades: SKIPPED: Cannot find "fwupdmgr" in PATH
Restarts: SKIPPED: Cannot find "needrestart" in PATH
 2021-02-15T08:15:43.342Z DEBUG topgrade::terminal   > Desktop notification: Topgrade finished sucessfully

</pre></details>

**Comment by r-darwish** [comment]
Should be okay now
